test/unit/component/parser/json/BpmnJsonParser.event-utils.ts

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright 2020 Bonitasoft S.A.
```

Comment:
**suggestion:** as these are new files, we could put 2023 instead of 2020
This remark applies to all files changed in this PR.

Reply:
```suggestion
Copyright 2023 Bonitasoft S.A.
```